doc/connectivity-semantics-and-api.md

```diff
@@ -34,14 +34,26 @@ retries are done with exponential backoff, channels that fail to connect will
 start out spending very little time in this state but as the attempts fail
 repeatedly, the channel will spend increasingly large amounts of time in this
 state. For many non-fatal failures (e.g., TCP connection attempts timing out
-because the server is not yet available), the channel may be stuck in this
-state for an indefinitely large amount of time.
-
-FATAL_FAILURE: There has been a fatal failure and the channel will never
-attempt to establish a connection again.  (e.g., a server presenting an invalid
-TLS certificate)
-
-Channels that enter this state never leave this state.
+because the server is not yet available), the channel may spend increasingly
+large amounts of time in this state.
+
+IDLE: This is the state where the channel is not even trying to create a
+connection because of a lack of new or pending RPCs. New channels MAY be created
```

Comment:
Synonyms. An active/pending RPC  is one that has been started but has not finshed (returned status). This is from the channel's perspective, so if the channel is wrapped by something and that wrapper buffers RPCs, those buffered RPCs will be active at the wrapper but not at the channel.